utils/contextutils/context.go

```diff
@@ -23,8 +29,20 @@ const (
 	// TimeReceivedMetadataKey is optional metadata in the gRPC response header that correlates
 	// to the time right after the point cloud was captured.
 	TimeReceivedMetadataKey = "viam-time-received"
+
+	readTimeout            = 5 * time.Second
```

Comment:
Now that we've moved these three `timeout` variables to context utils to avoid the import cycle, I think we'll need a) a small sentence describing what "read" they're governing (config reading) and b) to rename the variables to make it more clear what "read" they're governing.